eth_utils/hexadecimal.py

```diff
@@ -64,7 +69,17 @@ def is_hexstr(value: Any) -> bool:
         return False
 
     try:
-        value_as_bytes = codecs.decode(value_to_decode, "hex")  # type: ignore
+        # convert from a value like '09af' to b'09af'
+        ascii_hex = value_to_decode.encode("ascii")
+    except AttributeError:
```

Comment:
Just curious, why catch an `AttributeError` here?

Reply:
Really, just unnecessary caution, since there's already an `is_text()` check above. I agree and removed them in both places.